contributors/devel/sig-instrumentation/logging.md

```diff
@@ -39,7 +39,18 @@ As per the comments, the practical default level is V(2). Developers and QE
 environments may wish to run at V(3) or V(4). If you wish to change the log
 level, you can pass in `-v=X` where X is the desired maximum level to log.
 
-## Logging header formats
+## Logging Formats
+
+Kubernetes supports multiple logging formats to allow users flexibility in picking format that best matches their needs.
+As new feature and logging formats are proposed we expect that Kubernetes community might find itself in disagreement about what formats and features should be officially supported.
+This section will serve as a developer documentation describing high level goals of each format, giving a holistic vision of logging options and allow community to resolve any future disagreement when making changes.
+
+### Text logging format
+
+* Default format in Kubernetes.
+* Purpose: Maintain backward compatibility with klog format.
+* Human-readable, multiline support.
```

Comment:
I think human-readable is also a purpose, not just some accidental aspect of it. I would write it like this:
```
* Purpose:
  * Maintain backward compatibility with klog format.
  * Human-readable, with support for printing multi-line strings with actual line breaks.
```

Reply:
I will fix it in git. For future https://github.blog/changelog/2020-02-26-multi-line-code-suggestions-beta/